### DISCUSSION TITLE
Implement conditions for relations

### DISCUSSION THREAD

**Discussion by Robinvm** [filamentphp/filament#1087] [Ideas]
Hi Dan,

Can you implement the possibility to add conditions for relations in resources?

**Comment by ryangjchandler** [comment]
@Robinvm Could you provide an example of what you're trying to achieve?

**Comment by Robinvm** [comment]
I would like to show relationmanagers based on variables in the record

Met vriendelijke groet,
Robin van Maasakker
Directeur & lead development
Qubiqx
www.qubiqx.com<http://www.qubiqx.com/>
m: 06 - 38 74 65 85<tel:0638746585>
e:  ***@***.******@***.***>
l:  Bijsterhuizen 1158, 6546AS, Nijmegen<https://www.google.com/maps?sxsrf=ALeKk000s272mLcjQID9UalT5gMA5QYNsg:1614583778806&iflsig=AINFCbYAAAAAYDyl8sqPosX9KFX71MoQxfSC2DuKjUmf&q=Bijsterhuizen+1158,+6546AS,+Nijmegen&gs_lcp=Cgdnd3Mtd2l6EAMyAggmUIEFWIEFYLUJaABwAHgAgAHCAogBwgKSAQMzLTGYAQCgAQKgAQGqAQdnd3Mtd2l6&uact=5&um=1&ie=UTF-8&sa=X&ved=2ahUKEwiKosilyY7vAhVG-aQKHc_lAx0Q_AUoAXoECAUQAw>
[Afbeelding met tekening  Automatisch gegenereerde beschrijving]<https://qubiqx.com/>


Van: Ryan Chandler ***@***.***>
Datum: vrijdag, 31 december 2021 om 15:18
Aan: laravel-filament/filament ***@***.***>
CC: Robin van Maasakker | Qubiqx ***@***.***>, Mention ***@***.***>
Onderwerp: Re: [laravel-filament/filament] Implement conditions for relations (Discussion #1087)

@Robinvm<https://github.com/Robinvm> Could you provide an example of what you're trying to achieve?

**Comment by danharrin** [comment]
In the next version, you'll be able to override the `getRelationManagers()` method of the `Edit` page to customise which relation managers should be shown.

**Comment by Robinvm** [comment]
Amazing!

Met vriendelijke groet,
Robin van Maasakker
Directeur & lead development
Qubiqx
www.qubiqx.com<http://www.qubiqx.com/>
m: 06 - 38 74 65 85<tel:0638746585>
e:  ***@***.******@***.***>
l:  Bijsterhuizen 1158, 6546AS, Nijmegen<https://www.google.com/maps?sxsrf=ALeKk000s272mLcjQID9UalT5gMA5QYNsg:1614583778806&iflsig=AINFCbYAAAAAYDyl8sqPosX9KFX71MoQxfSC2DuKjUmf&q=Bijsterhuizen+1158,+6546AS,+Nijmegen&gs_lcp=Cgdnd3Mtd2l6EAMyAggmUIEFWIEFYLUJaABwAHgAgAHCAogBwgKSAQMzLTGYAQCgAQKgAQGqAQdnd3Mtd2l6&uact=5&um=1&ie=UTF-8&sa=X&ved=2ahUKEwiKosilyY7vAhVG-aQKHc_lAx0Q_AUoAXoECAUQAw>
[Afbeelding met tekening  Automatisch gegenereerde beschrijving]<https://qubiqx.com/>


Van: Dan Harrin ***@***.***>
Datum: zondag, 2 januari 2022 om 13:19
Aan: laravel-filament/filament ***@***.***>
CC: Robin van Maasakker | Qubiqx ***@***.***>, Mention ***@***.***>
Onderwerp: Re: [laravel-filament/filament] Implement conditions for relations (Discussion #1087)

In the next version, you'll be able to override the getRelationManagers() method of the Edit page to customise which relation managers should be shown.
